src/toggle-switch/toggle-switch.scss

```diff
@@ -2,194 +2,206 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
+$toggle-switch-height: $iui-baseline * 2;
+
 @mixin iui-toggle-switch {
   @include iui-reset;
-  display: flex;
+  display: grid;
+  grid-template: 'toggle';
+  grid-column-gap: $iui-s;
   align-items: center;
   font-size: $iui-font-size;
   width: fit-content;
-  padding: ($iui-baseline/2) 0;
+  padding: ($iui-baseline * 0.5) 0;
   user-select: none;
   cursor: pointer;
   position: relative;
+  isolation: isolate;
   @include themed {
     color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2));
   }
 
-  > input {
-    width: 0;
-    height: 0;
+  > input[type='checkbox'] {
+    @include iui-reset;
+    @include iui-focus($thickness: 1px, $offset: 1px);
+    grid-area: toggle;
+    display: inline-block;
+    flex-shrink: 0;
+    position: relative;
     appearance: none;
-    opacity: 0;
-    position: absolute;
+    width: $toggle-switch-height * 2 - $iui-xs;
```

Comment:
Why is width using this `-height` variable?

Reply:
I did some rethinking here.  Below is an image explaining what I'm trying to achieve spacing wise.
![Screen Shot 2022-02-04 at 1 19 26 PM](https://user-images.githubusercontent.com/849817/152582118-4838133b-513a-446f-a6b6-389104b81adb.png)

Before:  I was basing all the sizing off of the height of the toggle body.
Now:  I am basing all the sizing off of the toggle handle.

Everything scales properly based off the size set for the handle.  Here is what it looks like if the toggle handle is set to 24px instead of 16px.
![Screen Shot 2022-02-04 at 1 22 10 PM](https://user-images.githubusercontent.com/849817/152582450-3649d4f1-32c5-4b7f-b140-730cb346f0af.png)